### PR TITLE
kompose: remove travis-ci from branch protection

### PR DESCRIPTION
**Pull request by kadel**
We no longer use travis-ci for testing Kompose.

### CODE REVIEW

**Comment by chases2** [comment]
Can we get another maintainer of Kompose to LGTM this PR?

/assign

**Review by cdrage — APPROVED** [review]

**Comment by cdrage** [comment]
ping @chases2

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**
